packages/protocol-perf/src/index.ts

```diff
@@ -57,7 +57,14 @@ export const defaultInit: PerfServiceInit = {
 }
 
 export interface PerfService {
-  measurePerformance(startTime: number, connection: Connection, sendBytes: bigint, recvBytes: bigint, options?: AbortOptions): Promise<number>
+  measurePerformance(connection: Connection, sendBytes: bigint, recvBytes: bigint, options?: AbortOptions): Promise<PerfOutput>
```

Comment:
`BigInt` arithmetic is much slower than `Number` arithmetic in JavaScript - can `sendBytes` and `recvBytes` become `Number`s?

Reply:
We are currently testing single connection single stream throughput. Thus, I don't expect this benchmark to be CPU bound, but network bound. I don't expect the above optimization to make a difference. My intuition might be off.

Reply:
Either way the largest value we are accepting is [still of type Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) so it's a worthwhile refactor imo.